chat-core/src/integrations/openai.test.ts

```diff
@@ -1,49 +1,10 @@
-import {
-  OpenAiEmbeddingsClient,
-  OpenAiChatClient,
-} from "../../src/integrations/openai";
+import { OpenAiChatClient } from "../../src/integrations/openai";
```

Comment:
why are we just removing embedding stuff from openai integration but not also llm? 

perhaps future work

Reply:
Yeah, as big as this refactor is, the actual purpose is just to allow ingest to use embed. I left as much of the other stuff alone as I could.